internal/pkg/scaffold/ansible/dockerfilehybrid.go

```diff
@@ -92,7 +94,7 @@ COPY roles/ ${HOME}/roles/[[ end ]]
 [[- if .Playbook ]]
 COPY playbook.yml ${HOME}/playbook.yml[[ end ]]
 
-ENTRYPOINT ["/tini", "--", "bash", "-c", "${OPERATOR} run ansible --watches-file=/opt/ansible/watches.yaml $@"]
+ENTRYPOINT ["/tini", "--", "/usr/local/bin/entrypoint"]
```

Comment:
Should this be `/usr/local/bin/entrypoint run ansible --watches-file=/opt/ansible/watches.yaml $@` since entrypoint just runs the operator binary?

Reply:
Last line of entrypoint is:
`exec ${OPERATOR} run ansible --watches-file=/opt/ansible/watches.yaml $@`